doc/install.rst

```diff
@@ -134,7 +129,11 @@ This can happen if you have multiple versions of GMT installed.
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
 ``GMT_LIBRARY_PATH`` environment variable.
 This should be set to the directory where ``libgmt.so``, ``libgmt.dylib`` or ``gmt.dll``
-can be found for Linux, MacOS and Windows respectively.
+can be found for Linux, macOS and Windows respectively.
 e.g. in a terminal run::
 
-   export GMT_LIBRARY_PATH=$HOME/anaconda3/envs/pygmt/lib
+    # Linux/macOS
+    export GMT_LIBRARY_PATH=$HOME/anaconda3/envs/pygmt/lib
+    # Windows
+    set "GMT_LIBRARY_PATH=C:\Miniconda\envs\pygmt\Library\bin\"
```

Comment:
What about changing it to:
```suggestion
    set "GMT_LIBRARY_PATH=C:\Users\USERNAME\Anaconda3\envs\pygmt\Library\bin\"
```

Reply:
On Windows, Anaconda is installed to `C:\Users\<your-username>\Anaconda3\` by default.

Reply:
Either way, as long as there's an example. The Anaconda one does look better though.